variables.tf

```
@@ -167,37 +160,34 @@ variable "vm_auto_healing_enabled" {
   type        = bool
 }
 # TFE VARS
-variable "license_secret" {
-  description = <<-EOD
-  The Secret Manager secret which comprises the Base64 encoded Replicated license file. The Terraform provider calls
-  this value the secret_id and the GCP UI calls it the name.
-  EOD
-  type        = string
-}
-variable "fqdn" {
-  description = "Fully qualified domain name for the TFE endpoint"
+variable "airgap_url" {
+  default     = null
+  description = "The URL of the storage bucket object that comprises an airgap package."
   type        = string
 }
-variable "ssl_certificate_name" {
+variable "ca_certs" {
   default     = null
-  description = "Name of the created managed SSL certificate. Required when load_balancer == \"PUBLIC\" or load_balancer == \"PRIVATE\"."
+  description = <<-EOD
+  A custom Certificate Authority certificate bundle to be used for authenticating connections with Terraform Enterprise.
+  EOD
   type        = string
 }
-
-variable "ssl_certificate_secret" {
+variable "extra_no_proxy" {
   default     = null
   description = <<-EOD
-  The Secret Manager secret which comprises the Base64 encoded PEM certificate file. The Terraform provider calls this
-  value the secret_id and the GCP UI calls it the name. This value is only used when load_balancer == "PRIVATE_TCP".
+  A list of hosts for which Terraform Enterprise will not use a proxy to access. The list must be a comma-separated
+  string, like \".example.com,.example.org\".
   EOD
   type        = string
 }
```

This will also need to be adjusted to match the inner variable:

```suggestion
variable "extra_no_proxy" {
  default     = []
  description = "A list of hosts for which Terraform Enterprise will not use a proxy to access."
  type        = list(string)
}
```